stable/netsil/values.yaml

```diff
@@ -0,0 +1,9 @@
+deployment:
+  image: netsil/netsil
+  pullPolicy: ifNotPresent
+  tag: stable-1.5.3
```

Comment:
Please add more config options, e. g. pod annotations, resources, node selectors. I'd suggest you do it like so:

```yaml 
streamProcessor:
  image:
    repository: netsil/netsil
    tag: stable-1.5.3
    pullPolicy: IfNotPresent
  
  service:
    annotations: {}
    clusterIP: ""
    type: NodePort

  podAnnotations: {} 

  resources: {}

  nodeSelector: {}

collector:
  image:
    repository: netsil/collectors
    tag: stable-1.5.3
    pullPolicy: IfNotPresent
  
  podAnnotations: {} 

  resources: {}

  nodeSelector: {}
```

Note: `ifNotPresent` -> `IfNotPresent`

Reply:
Is this nice to have or a requirement?

Reply:
fixed.

Reply:
Well, didn't add the config options. :)

Reply:
Treating that as a nice to have given I don't have too much time to spend on this Chart.